README.md

```diff
@@ -1,40 +1 @@
-
-![Alt Text](https://aka.ms/bridge-to-k8s-graphic-non-isolated)
-
-### Key Features:
-
-#### Simplifying Microservice Development 
-- Eliminate the need to manually source, configure and compile external dependencies on your development computer.  
-
-#### Easy Debugging 
-- Run your usual debug profile with the added cluster configuration. You can debug your code as you normally would while taking advantage of the speed and flexibility of local debugging. 
-
-#### Developing and Testing End-to-End 
-- Test end-to-end during development time. Select an existing service in the cluster to route to your development machine where an instance of that service is running locally. Request initiated through the frontend of the application running in Kubernetes will route between services running in the cluster until the service you specified to redirect is called. 
-
-## Documentation
-- [Visual Studio](https://aka.ms/bridge-to-k8s-vs-quickstart)
-- [Visual Studio Code](https://aka.ms/bridge-to-k8s-vscode-quickstart)
-- [How Bridge to Kubernetes Works](https://aka.ms/how-bridge-to-k8s-works)
-
-## Roadmap
-https://github.com/microsoft/mindaro/projects
-
-## Purpose of this repository
-This source repository primarily hosts *code samples* to support product guides, discussion of bugs/feature requests, as well as provide high-level insight into our product roadmap.
-
-## Contributing
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
-
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+**NOTICE:** Bridge to Kubernetes code base is being made open source and this sample repo is being archived in favor of [new repo](https://github.com/Azure/Bridge-To-Kubernetes) with actual code and samples. Microsoft devs are actively developing and monitoring that repo, please feel free to rase any issue or feature request there. New repo will be updated with roadmap in hte near future.
```

Comment:
Minor comments:
* `Microsoft devs are actively developing and monitoring that repo` => I'd consider saying `the new repository` or something, so that it's not ambiguous which one we're talking about.
* I don't have the guidelines in mind, so I might misremember, but I believe that ideally we try to use full words (developers, repository, etc.) when possible.
* Typos: `feel free to rase` => `raise` and `in hte near future` => `the`